### PR TITLE
WT-12457 Huffman encoding: remove docs

### PR DESCRIPTION
**Pull request by korteland**
This also includes a fix for me not realising `test/format/config.sh` needs to be manually run when removing a config item.

### CODE REVIEW

**Comment by korteland** [comment]
Note that `rec_col.c` still contains a reference to Huffman coding - that unpack hides another bug with overflow items, so I'm going to do that in a separate ticket.

File: src/docs/file-formats.dox [hunk 1]
```diff
@@ -117,8 +108,7 @@ considered.   (See @ref compression_considerations for details.)
   Block compression is disabled by default.
 
 Variable-length column stores support four types of compression:
```
Comment (tod-johnson-mongodb) [inline src/docs/file-formats.dox:110]: Change "four" to "three". Alternately change it to "several" to keep it correct if we change the compression types again.

**Review by tod-johnson-mongodb — COMMENTED** [review]
There is a one word error: "four" instead of "three" or "several".  Therefore not approved yet. Aside from that it LGTM.

File: src/docs/file-formats.dox (hunk 1 above)
Comment (korteland) [reply src/docs/file-formats.dox]: Good catch - thanks! Fixed in 714ae27.

**Review by ruby-oujo — APPROVED** [review]

**Review by tod-johnson-mongodb — APPROVED** [review]
LGTM

**Pull request merged**